### PR TITLE
EXP-1283 Add twitter URL to the location SDK

### PR DESCRIPTION
**Pull request by Varrah**

### CODE REVIEW

File: WellnessLiving/Wl/Location/View/ViewModel.php [hunk 1]
```diff
@@ -309,6 +309,13 @@ class ViewModel extends WlModelAbstract
    */
   public $url_site = '';
 
+    /**
+     * Twitter url.
+     *
+     * @var string
+     */
+    public string $url_twitter = '';
```
Comment (GreggHenry02) [inline WellnessLiving/Wl/Location/View/ViewModel.php:317]: Same as the last one :)

File: WellnessLiving/Wl/Location/View/ViewModel.php [hunk 2]
```diff
@@ -309,6 +309,13 @@ class ViewModel extends WlModelAbstract
    */
   public $url_site = '';
 
+    /**
+     * Twitter url.
+     *
+     * @var string
```
Comment (GreggHenry02) [inline WellnessLiving/Wl/Location/View/ViewModel.php:315]: Is this returned as part of the get? It should have the '@get result' tag then.

File: WellnessLiving/Wl/Location/View/ViewModel.php [hunk 3]
```diff
@@ -309,6 +309,13 @@ class ViewModel extends WlModelAbstract
    */
   public $url_site = '';
 
+    /**
+     * Twitter url.
```
Comment (GreggHenry02) [inline WellnessLiving/Wl/Location/View/ViewModel.php:313]: Which twitter URL? Probably the URL of the location's twitter account, it should be said.

**Review by GreggHenry02 — CHANGES_REQUESTED** [review]

File: WellnessLiving/Wl/Location/View/ViewModel.php (hunk 1 above)
Comment (Varrah) [reply WellnessLiving/Wl/Location/View/ViewModel.php]: Fixed

File: WellnessLiving/Wl/Location/View/ViewModel.php (hunk 2 above)
Comment (Varrah) [reply WellnessLiving/Wl/Location/View/ViewModel.php:316]: Fixed

File: WellnessLiving/Wl/Location/View/ViewModel.php (hunk 3 above)
Comment (Varrah) [reply WellnessLiving/Wl/Location/View/ViewModel.php]: Copied that over from the corresponding API block. Now fixed.

**Pull request merged**